examples/server_container.toml

```diff
@@ -81,6 +81,10 @@ domain = "idm.example.com"
 #   origin = "https://idm.example.com"
 origin = "https://idm.example.com:8443"
 #
+# Allowed CORS origins. This allows deployment of an
```

Comment:
I wonder how this will play with webauthn or cookies? Since we need both to be related to origin, shouldn't we just set cors to origin and subdomains of origin instead? Alternately we need to enforce all the allow_origins are subdomains of origin.

Reply:
I wouldn't note it as an alternative webUI for Kanidm, more "allow access to the Kanidm endpoints from additional domains".

Origin's already covered as it's not a cross-origin request, but I'm not sure about sub/parent domains (I'm pretty sure they'd be blocked)

This needs more documentation in the book